Source/Core/Core/PowerPC/JitCommon/ConstantPropagation.cpp

```diff
@@ -371,6 +371,9 @@ ConstantPropagationResult ConstantPropagation::EvaluateTable31SB(UGeckoInstructi
 
   switch (inst.SUBOP10)
   {
+  case 24:  // slwx
+    a = u32(u64(s) << b);
```

Comment:
```suggestion
    a = u32(u64(s) << (b & 0x3f));
```
To avoid undefined behavior if b >= 64.